src/components/Filters/AsideFilter.tsx

```diff
@@ -1,12 +1,47 @@
+import { useEffect, useState } from "react";
+import { BrandFilter } from "./BrandFilter";
+import { CategoryFilter } from "./CategoryFilter";
 import { PriceFilter } from "./PriceFilter";
-import { RatingFilter } from "./RaitingFilter";
+import { RatingFilter } from "./RatingFilter";
+import { PriceRange } from "./types";
+import axios from "axios";
+import { url } from "../../main/constants/common";
 
 export const AsideFilter = () => {
+  const [priceRange, setPriceRange] = useState<PriceRange>({
+    min: 0,
+    max: 1000,
+  });
+  const [availableRange, setAvailableRange] = useState<PriceRange>({
+    min: 0,
+    max: 1000,
```

Comment:
It has to be dynamic. Code duplicate can be in the separate constant

Reply:
done